### ISSUE TITLE
Discourage Inventory Profiles Next

### ISSUE THREAD

**Issue by Technici4n** [AppliedEnergistics/Applied-Energistics-2#8101]
https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/7834
https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/7920

**Comment by blackd** [comment]
Can't we just try and make the mods work better together?

**Comment by blackd** [comment]
Yes but next time you decide to change some package name it will break again. Since it's a static json file.
And apparently the problem was shift+click not the sorting.

**Comment by blackd** [comment]
And I don't want to shift the blame. Instead I hoped it will be possible to get together and see what the problems are and how to fix them.
This will lead to more stable solution instead of just hoping the other side doesn't make breaking changes.

**Comment by Mari023** [comment]
newer Inventory Profiles Next version ship with a default configuration which should fix both linked issues, imo this can be closed.
